group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -843,15 +886,19 @@ public JoinGroupResponseData setupGroupWithPendingMember(GenericGroup group) thr
             // Start the join for the second member
             CompletableFuture<JoinGroupResponseData> followerJoinFuture = new CompletableFuture<>();
             CoordinatorResult<Void, Record> result = sendGenericGroupJoin(
-                joinRequest.setMemberId(UNKNOWN_MEMBER_ID),
+                joinRequest
+                    .setMemberId(UNKNOWN_MEMBER_ID),
                 followerJoinFuture
             );
 
             assertTrue(result.records().isEmpty());
             assertFalse(followerJoinFuture.isDone());
 
             CompletableFuture<JoinGroupResponseData> leaderJoinFuture = new CompletableFuture<>();
-            result = sendGenericGroupJoin(joinRequest.setMemberId(leaderId), leaderJoinFuture);
+            result = sendGenericGroupJoin(
+                joinRequest
+                    .setMemberId(leaderJoinResponse.memberId()),
+                leaderJoinFuture);
```

Comment:
nit: I have noticed that we are not consistent about where we put the closing parenthesis.